src/main/java/interface_adapter/leaderboard/LeaderboardPresenter.java

```diff
@@ -28,28 +29,34 @@ public LeaderboardPresenter(LeaderboardViewModel leaderboardViewModel, ViewManag
      */
     @Override
     public void prepareSuccessView(LeaderboardOutputData outputData) {
+        LeaderboardState leaderboardState = leaderboardViewModel.getState();
+
         Map<Integer, CommonUser> topUsers = outputData.getTopUsers();
         for(Integer rank : topUsers.keySet()) {
             CommonUser currentUser = topUsers.get(rank);
             if(currentUser.getName().equals(outputData.getCurrentUsername())){
-                System.out.println("Were looking at you rn (current logged in)");
-                LeaderboardViewModel.setCurrentUserStats(rank, outputData.getCurrentUsername(), currentUser.getAveragePoints() );
+
+                leaderboardState.setCurrentUserName(outputData.getCurrentUsername());
+                leaderboardState.setCurrentUserAvgPoints(currentUser.getAveragePoints());
+
             }
             switch(rank) {
-                default:
```

Comment:
I think we should keep the default in switch since checkstyle and solar lint ask for it